pkg/reconciler/pipelinerun/cancel_test.go

```diff
@@ -97,56 +109,275 @@ func TestCancelPipelineRun(t *testing.T) {
 			{ObjectMeta: metav1.ObjectMeta{Name: "t2"}},
 		},
 	}, {
-		name: "deprecated-state",
+		name:           "deprecated-state",
+		embeddedStatus: config.DefaultEmbeddedStatus,
 		pipelineRun: &v1beta1.PipelineRun{
 			ObjectMeta: metav1.ObjectMeta{Name: "test-pipeline-run-cancelled"},
 			Spec: v1beta1.PipelineRunSpec{
 				Status: v1beta1.PipelineRunSpecStatusCancelledDeprecated,
 			},
 		},
+	}, {
+		name:           "child-references",
+		embeddedStatus: config.BothEmbeddedStatus,
```

Comment:
could you please add a test case for minimal embedded status?

Reply:
👍